src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothEventArgs.cs

```diff
@@ -560,6 +560,46 @@ public IBluetoothServerSocket Server
         }
     }
 
+    /// <summary>
+    /// An extended EventArgs class contains the socket connection requested.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class SocketConnectionRequestedEventArgs : EventArgs
+    {
+        private int _socketFd;
+        private string _remoteAddress;
+
+        internal SocketConnectionRequestedEventArgs(int socketFd, string remoteAddress)
+        {
+            _socketFd = socketFd;
+            _remoteAddress = remoteAddress;
+        }
+
+        /// <summary>
+        /// The socket fd.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public int SocketFd
```

Comment:
What can I do with SocketFD in C#?

Reply:
SocketFd is too low level to use in c# 
How about change to more high level object?

Reply:
As you know, 'SocketFd' is stored internally.
So, no need to provide it to user. I will remove it on next commit.

Reply:
Please change to internal

Reply:
Done. Thank you.